### ISSUE TITLE
Memory leak when playing video on firefox browser

### ISSUE THREAD

**Issue by danyjf** [mrdoob/three.js#26527]
### Description

When opening a three.js example from the three.js website on firefox (version 116.0) the memory usage shown on task manager keeps increasing.

### Reproduction steps

1. Open firefox
2. Go to one of the three.js video examples
3. Open task manager
4. Look at the firefox memory usage

### Code

The code is in the three.js examples. 
From my tests the leak happens when rendering a mesh with a material that uses a THREE.VideoTexture as the map parameter.

### Live example

* [webgl materials video](https://threejs.org/examples/?q=video#webgl_materials_video)
* [webgl video panorama](https://threejs.org/examples/?q=video#webgl_video_panorama_equirectangular)

### Screenshots

Screenshots from playing video on [this example](https://threejs.org/examples/?q=video#webgl_materials_video).

Memory before playing video:
![image](https://github.com/mrdoob/three.js/assets/76452190/67bcda8a-7755-4303-bd04-80dbd93ca236)

Memory after 2 minutes of playing video:
![image](https://github.com/mrdoob/three.js/assets/76452190/fb6222bf-767d-45f8-beb0-ddc837638b1a)

Memory after 5 minutes of playing video:
![image](https://github.com/mrdoob/three.js/assets/76452190/484e19ca-c6f8-42b9-a144-1e5ed487cfad)

### Version

r155

### Device

Desktop

### Browser

Firefox

### OS

Windows

**Comment by Mugen87** [comment]
There was a recent change in `dev` in context of video textures. Do you mind checking if the memory leak happens with the following URL as well:

https://rawcdn.githack.com/mrdoob/three.js/dev/examples/webgl_video_panorama_equirectangular.html

**Comment by Mugen87** [comment]
@danyjf Do you mind reporting this issue to Mozilla at https://bugzilla.mozilla.org/ ?

I'm on macOS and could not reproduce the memory leak with the above examples. When testing `webgl_video_panorama_equirectangular` the memory of the tab stays around 75MB. 

<img width="1985" alt="image" src="https://github.com/mrdoob/three.js/assets/12612165/8e84aa3f-7a0a-4215-9c8f-1e9bcdc8ac00">

So maybe this issue is Windows exclusive.

**Comment by danyjf** [comment]
I was traveling when I saw this comment and only had access to my laptop, but when I tested on my laptop (which is also running windows 11 with same version of Firefox), the memory leak didn't happen.

When I got back home tested again on my desktop PC and the memory leak was still happening on it.
Decided to try updating the graphics drivers and that seems to have fixed it for me!

**Graphics card**: AMD Radeon RX 6700 XT.
**Old driver version**: 22.40.57.02 Adrenalin Edition *(Most recent version at the beginning of January 2023 when I installed it)*
**Updated driver version**: 23.10.23.02 Adrenalin Edition *(The most recent version available as of 17 August 2023)*
